spark-indexer/src/main/java/org/uniprot/store/spark/indexer/main/verifiers/VerifyCitationData.java

```diff
@@ -0,0 +1,188 @@
+package org.uniprot.store.spark.indexer.main.verifiers;
+
+import static org.uniprot.store.spark.indexer.common.util.SparkUtils.getInputReleaseDirPath;
+
+import java.util.ResourceBundle;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.api.java.function.Function;
+import org.uniprot.core.citation.Citation;
+import org.uniprot.core.citation.CitationDatabase;
+import org.uniprot.core.uniprotkb.UniProtKBReference;
+import org.uniprot.core.util.Pair;
+import org.uniprot.core.util.PairImpl;
+import org.uniprot.store.spark.indexer.common.JobParameter;
+import org.uniprot.store.spark.indexer.common.util.SparkUtils;
+import org.uniprot.store.spark.indexer.literature.mapper.LiteratureFileMapper;
+
+/**
+ * @author lgonzales
+ * @since 22/03/2021
+ */
+@Slf4j
+public class VerifyCitationData {
+
+    private static final String SPLITTER = "\n//";
+
+    public static void main(String[] args) {
+        ResourceBundle config = SparkUtils.loadApplicationProperty();
+        JavaSparkContext sparkContext = SparkUtils.loadSparkContext(config);
+
+        JobParameter jobParameter =
+                JobParameter.builder()
+                        .applicationConfig(config)
+                        .releaseName("2021_02")
+                        .sparkContext(sparkContext)
+                        .build();
+        // cat medaaa | grep '^//\w' to find --> problem in the file  //MuRF2
+        for (String file : args) {
+            log.info("---------------- Literature File Name: {} ---------------------", file);
+            String releaseInputDir = getInputReleaseDirPath(config, jobParameter.getReleaseName());
+            String literaturePath = releaseInputDir + "literature/" + file;
+            sparkContext.hadoopConfiguration().set("textinputformat.record.delimiter", SPLITTER);
+            long count =
+                    sparkContext
+                            .textFile(literaturePath, 1000)
+                            .mapToPair(new LiteratureFileMapper())
+                            .count();
+            log.info("Number of entries: {}", count);
+            /*            List<Tuple2<String, Literature>> literature = sparkContext.textFile(literaturePath, 1000)
```

Comment:
If we don't need the below commented code please delete it

Reply:
This was a test class. I deleted it